contents/docs/settings/access-control.mdx

```diff
@@ -65,48 +67,95 @@ You can override the default access level for specific members or [roles](#role-
 
-Access control is available for:
+Currently, resource access controls are available for:
 - Insights
 - Dashboards
 - Notebooks
 - Feature flags
-- (more resource types coming soon)
+- (more resource types coming soon – looking for others? Let us know!)
 
-Each resource has three possible access levels:
-- **No access** - Cannot view or edit the resource
-- **View** - Can view but not modify the resource
-- **Edit** - Can view and modify the resource
+Note: We do not yet support limiting access to querying data, viewing replays, or accessing people data. Support for these features is planned for the near future.
 
-Resources default to "Edit" access when created. You can modify the default and set specific permissions for members and roles.
+Resource access controls have three possible access levels:
+- **No access** – Cannot view or edit the resource
+- **View** – Can view but not modify the resource
+- **Edit** – Can view and modify the resource
 
-Resource creators and project admins can always view and edit resources as well as manage access controls.
+There are two ways to set resource-level access controls:
 
-## Role-based access control (RBAC)
+#### a. Individual resource object
 
-> RBAC is only available on Enterprise plans
+These settings allow you to control who can view and edit a specific resource object. You can access these controls via the project's access control settings.
 
-Instead of managing permissions individually, you can create roles to group users together. Roles can be assigned permissions at both the project and resource level.
+By default, new resources are set to "Edit" access. Users with appropriate permissions can modify this default and set specific permissions for members and roles.
+
+Resource creators and project admins can always view and edit resources, as well as manage their access controls. Only creators and project admins can manage access controls for a resource object.
+
+You cannot set resource-level access controls for project admins, as they always have full access.
 
 <ProductScreenshot
-    imageLight = {rbacSettingsLight} 
-    imageDark = {rbacSettingsDark}
-    alt="RBAC settings" 
+    imageLight = {objectAccessControlLight} 
+    imageDark = {objectAccessControlDark}
+    alt="Object access control" 
+    classes="rounded"
+/>
+
+#### b. All resource objects of a given type in a project
+
+These settings allow you to control who can view and edit all resources of a given type within a project. These controls are set at the project level.
+
+You can set default access levels for all resources of a given type in a project. This allows you to set it once and apply it to all resources of that type in the project (past and future).
+
+Project-wide access controls for resources take precedence over individual resource object access controls.
```

Comment:
This is maybe a bit counterintuitive, I would've expected project-wide defaults that can be overridden on a resource level, though I understand this was probably the rationale for it?

> This allows you to set it once and apply it to all resources of that type in the project (past and future)